### PR TITLE
{tools}[GCCcore/12.3.0] Jupyter-bundle v20230823, jupyter-resource-usage v1.0.0, jupyterlmod v4.0.3

### PR DESCRIPTION
**Pull request by casparvl**
(created using `eb --new-pr`)

- [x] Depends on #18617

### CODE REVIEW

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/9212ee0c66b83089d79a34e2ecfb4b4b for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/82e4be6ed42705a74b558dbe7c8c0d68 for a full test report.

**Comment by verdurin** [comment]
@casparvl please note the test error about missing `sanity_check_paths` in `Jupyter-bundle`.

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/c4d7cc08b910d9d4d79b0e8840d63067 for a full test report.

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen2

**Comment by casparvl** [comment]
Someone at some point explain to me how I can combine a `--from-pr` with `--update-pr` without pushing in a whole bunch of EasyConfigs again in the update that were in the `--from-pr`...

Anyway, should be updated for https://github.com/easybuilders/easybuild-easyconfigs/pull/18735

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/6cc1af05c182a3eac82d94ac2364558d for a full test report.

File: easybuild/easyconfigs/j/Jupyter-bundle/Jupyter-bundle-20230823-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+dependencies = [
+    ('JupyterHub', '4.0.2'),
+    ('JupyterLab', '4.0.5'),
+    ('JupyterNotebook', '7.0.2'),
+    ('nbclassic', '1.0.0'),
+    ('jupyter-server-proxy', '4.0.0'),
+    ('jupyterlmod', '4.0.3'),
+    ('jupyter-resource-usage', '1.0.0'),
+]
```
Comment (boegel) [inline easybuild/easyconfigs/j/Jupyter-bundle/Jupyter-bundle-20230823-GCCcore-12.3.0.eb:24]: @casparvl Please add a `moduleclass` here (why the hell doesn't the easyconfigs test suite trip on that, huh...)

File: easybuild/easyconfigs/j/jupyter-resource-usage/jupyter-resource-usage-1.0.0-GCCcore-12.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/jupyter_resource_usage'],
+}
+
+# Add the notebook extension to the search path for jupyter notebooks
+modextrapaths = {'JUPYTER_PATH': 'share/jupyter/'}
```
Comment (boegel) [inline easybuild/easyconfigs/j/jupyter-resource-usage/jupyter-resource-usage-1.0.0-GCCcore-12.3.0.eb:35]: add `share/jupyter` to `dirs` in `sanity_check_paths`?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/j/Jupyter-bundle/Jupyter-bundle-20230823-GCCcore-12.3.0.eb (hunk 1 above)
Comment (casparvl) [reply easybuild/easyconfigs/j/Jupyter-bundle/Jupyter-bundle-20230823-GCCcore-12.3.0.eb:24]: Done

File: easybuild/easyconfigs/j/jupyter-resource-usage/jupyter-resource-usage-1.0.0-GCCcore-12.3.0.eb (hunk 2 above)
Comment (casparvl) [reply easybuild/easyconfigs/j/jupyter-resource-usage/jupyter-resource-usage-1.0.0-GCCcore-12.3.0.eb:38]: Done

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/5b62c8928d084b973ee0c297e9d69e38 for a full test report.

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/cfb723a7f3af43add51f09d98e2cbb93 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @casparvl!

**Pull request merged**